distribution/lib/Standard/Test/0.0.0-dev/src/Extensions.enso

```diff
@@ -296,7 +296,7 @@ Number.should_equal : Float -> Float -> Integer -> Spec_Result
 Number.should_equal self that epsilon=0 frames_to_skip=0 =
     matches = case that of
         _ : Number -> self.equals that epsilon
-        _ -> False
+        _ -> self==that
```

Comment:
9f4cde3d9f620d31820f736a6796d4e9990ea168 - otherwise `Number` couldn't be `should_equal` to `Complex`.